lib/SPIRV/LLVMToSPIRVDbgTran.h

```diff
@@ -145,6 +145,10 @@ class LLVMToSPIRVDbgTran {
   template <class T> SPIRVExtInst *getSource(const T *DIEntry);
   SPIRVEntry *transDbgFileType(const DIFile *F);
 
+  // Split Debug information
```

Comment:
Not sure what is meant by 'split debug information'. Can you pls explain?

Thanks

Reply:
From https://github.com/KhronosGroup/SPIRV-Registry/blob/main/nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc#DebugBuildIdentifier:

DebugStoragePath is a hint for consumers as to where to store this shader’s debug information. If the debug information has been split apart and is identified with [DebugBuildIdentifier](https://github.com/KhronosGroup/SPIRV-Registry/blob/main/nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc#DebugBuildIdentifier), this path can provide a hint as to where the debug information has been stored.

Perhaps updating the comment to:
// Generate DebugInfo Instructions recording where debug information was split to